app/views/i_belong_mailer/auto_enrolled.html.erb

```diff
@@ -15,37 +16,38 @@
     <strong>Next steps</strong>
   </p>
   <p class='govuk-body'>
-    Did you know that completing the Encouraging girls into GCSE computer science course means you're already on the way to achieving the I Belong certificate for your school? 
+    Did you know that completing the Encouraging girls into GCSE computer science course means you're already on the way to achieving the I Belong certificate for your school?
   </p>
   <p class='govuk-body'>
-    Now you're enrolled, you will be able to access your personal dashboard, where you can find out more about the certificate and the next steps required to achieve it.  
+    Now you're enrolled, you will be able to access your personal dashboard, where you can find out more about the certificate and the next steps required to achieve it.
   </p>
 
   <p class='govuk-body'>
     <%= link_to 'Explore activities in your dashboard', i_belong_url, class: 'govuk-button button' %>
   </p>
 
   <p class='govuk-body'>
-    Has your school already completed some of the activities required to achieve the certificate? Tick them off by submitting evidence in your dashboard.  
+    Has your school already completed some of the activities required to achieve the certificate? Tick them off by submitting
+    <%= link_to 'evidence', static_asset_url('Structuring+your+evidence+for+I+Belong.docx') %> in your dashboard.
```

Comment:
Can we swap this to a helper method pointing at 
https://media.teachcomputing.org/Structuring_your_I_Belong_evidence_7f7a83c21e.pdf